bw2io/extractors/ecospold2.py

```diff
@@ -57,6 +86,29 @@ def extract_metadata(o):
 
     @classmethod
     def extract(cls, dirpath, db_name, use_mp=True):
+        """
+        Extract data from all ecospold2 files in a directory.
+
+        Parameters
+        ----------
+        dirpath : str
+            The path to the directory containing the ecospold2 files.
+        db_name : str
+            The name of the database to create.
+        use_mp : bool, optional
+            Whether to use multiprocessing to extract the data (default is True).
```

Comment:
Here you use a different way to describe `default=True` (compare https://github.com/brightway-lca/brightway2-io/pull/176/files#r1157006084). Check which one is recommended and use that.

Reply:
Revised.

Reply:
Here you use `(default is True)`, [here](https://github.com/brightway-lca/brightway2-io/pull/176/files#r1157006084) you use `by default True` - which one is recommended?

Reply:
I changed this to "by default True." Don't really know which is recommended, but they used it like this in https://github.com/sphinx-contrib/napoleon/blob/dce30797b7a229ccebda4030f65482d501427794/docs/source/example_numpy.py#L226.

Reply:
We are following the numpy documentation style, which recommends [following PEP 257](https://peps.python.org/pep-0008/#documentation-strings), which in turn [uses `(default 0.0)`](https://peps.python.org/pep-0257/). I don't love this but its better to be consistent.